crates/polars-core/src/chunked_array/ops/sort/mod.rs

```diff
@@ -159,6 +159,33 @@ macro_rules! sort_with_fast_path {
     }}
 }
 
+macro_rules! arg_sort_fast_path {
```

Comment:
No blocker for this PR, but I rather see this in a generic function. Could be a follow up.